src/routes/write-exercise/__tests__/WriteExerciseScreen.spec.tsx

```diff
@@ -69,12 +69,13 @@ describe('WriteExerciseScreen', () => {
     }
   }
 
-  it('should show network error', () => {
-    mockUseLoadAsyncWithError('Network Error')
-    const { findByText } = render(<WriteExerciseScreen route={route} navigation={navigation} />, {
+  it('should show network error', async () => {
+    const errorMessage = 'Network Error'
+    mockUseLoadAsyncWithError(errorMessage)
+    const { findByText, debug } = render(<WriteExerciseScreen route={route} navigation={navigation} />, {
       wrapper: wrapWithTheme
     })
-    expect(findByText(labels.general.error.noWifi)).toBeDefined()
+    expect(await findByText(`${labels.general.error.noWifi} (${errorMessage})`)).toBeDefined()
```

Comment:
Do we really want to display the error message as well?

Reply:
Hmm. That's how it's implemented currently. It's basically the same as the other comment. This test shouln't have worked beforehand.